jetbrains-core-gui/tst/software/aws/toolkits/jetbrains/ui/s3/S3BrowserTest.kt

```diff
@@ -148,6 +155,11 @@ class S3BrowserTest : EmptyProjectTestCase() {
 
     @After
     fun cleanUp() {
+        // TODO fix tests on 2019.3
+        val info = ApplicationInfo.getInstance()
+        if (info.majorVersion == "2019" && info.minorVersionMainPart == "2") {
```

Comment:
the comment says 2019.3 but the code checks for "2" . is that intended?

Reply:
Yep, this was unfortunately because it used to be an `Assume.true`, fixed